src/pb/FDoper.cc

```diff
@@ -467,6 +467,108 @@ void FDoper<T>::del2_4th(GridFunc<T>& A, GridFunc<T>& B) const
 
     del2_4th_tm_.stop();
 }
+
+template <class T>
+void FDoper<T>::del2_4th(const Grid& Agrid, T* A, T* B, const size_t nfunc) const
```

Comment:
This function is a generalization of the previous one, with an extra parameter "nfunc". Is there a reason to keep the previous function, or should we now use that one with "nfunc=1" instead?

Reply:
This is a good idea. Let me replace the previous 3-D one in another PR. The only thing is now only the implementation of the 4th-order scheme is changed for OMP offload. What about the rest of the schemes?

Reply:
To 'replace it' I mean if the argument of apply() is a single vector, the function will pass nfunc=1 and call the new 4-D implementation.